src/MechanismGeometries.jl

```diff
@@ -1,17 +1,18 @@
-__precompile__(true)
+VERSION < v"0.7.0-beta2.199" && __precompile__()
```

Comment:
This should (luckily) no longer be necessary on 0.7 (i.e., you can just leave the `__precompile__(true)`). Ref https://github.com/JuliaLang/julia/pull/28459.